content/en/docs/Writing policies/background.md

```diff
@@ -5,9 +5,9 @@ description: >
 weight: 9
 ---
 
-Kyverno applies policies during admission control and to existing resources in the cluster that may have been created before a policy was created. The application of policies to existing resources is referred to as `background scanning`. 
+Kyverno can validate existing resources in the cluster that may have been created before a policy was created. The application of policies to existing resources is referred to as `background scanning`. 
 
-Note, that Kyverno does not mutate existing resources during scans, and will only report policy violations for existing resources that do not match policy rules.
+**Note: Kyverno does not mutate existing resources, to prevent inadvertent changes to workloads. Mutate and generate rules are not processed during background scans.** 
```

Comment:
Should we also note that violations are reported on mutate policies before v1.3.0?

Reply:
I suspect that may confuse most readers....